units/XNC0001/XNC0001_script.lua

```diff
@@ -1,10 +1,12 @@
--- The surface support vehicle that's in orbit
-
 local NCivilianStructureUnit = import('/lua/nomadsunits.lua').NCivilianStructureUnit
 local NCommandFrigateUnit = import('/lua/nomadsunits.lua').NCommandFrigateUnit
 
-
+--- Civilian
```

Comment:
Isn't this the vehicle following the ACU from orbit? It's not really a civilian entity then.